README.md

```diff
@@ -65,6 +67,39 @@ caffeinate -s pipenv run \
                 --app_gen_output_dir "$HOME/Desktop/multisuite_build_results/app_gen"
 ```
 
+You may also generate a project that matches your own project's dependency graph by using `--gen_type dot` parameter as well as supplying the location of the `dot` file that represents the graph:
+
+```bash
+pipenv run ./genproj.py --output_directory "$HOME/Desktop/mockapp" \
+                        --project_generator_type "cocoapods" \
+                        --gen_type dot \
+                        --dot_file_path "$HOME/MyProject/my_project_graph.dot" \
+                        --dot_root_node_name "MyProject" \
+                        --lines_of_code 150000
+```
+
+Additionally, you may also supply an optional JSON file to be used as a LOC map.  This allows you to generate a project from your own dependency graph in which each generated module has proportional LOC to your original graph.
```

Comment:
Could you give an example `cloc` or `tokei` command on how to do so here?  Or did you do something else to generate the json file?

Reply:
will do